From 0c98ac31fd550a2c8e24ea99cbc892d163f64026 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 27 May 2010 15:17:20 +0400
Subject: [PATCH 1/9] Force init script installation

---
 configure.ac |   16 +++-------------
 1 files changed, 3 insertions(+), 13 deletions(-)

diff --git a/configure.ac b/configure.ac
index 679c6ca..f2a9f7e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -295,19 +295,9 @@ launchd_enabled=false
 
 if test "$use_init" = "yes"; then
    AC_MSG_CHECKING(location of init directory)
-   if test -d /etc/rc.d; then
-       init_enabled=true
-       AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
-       AC_MSG_RESULT(${initdir})
-   else
-       if test -d /etc/init.d; then
-           init_enabled=true
-           AC_SUBST([initdir], ['${sysconfdir}/init.d'])
-           AC_MSG_RESULT(${initdir})
-       else
-           AC_MSG_RESULT(not found)
-       fi
-    fi
+   init_enabled=true
+   AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
+   AC_MSG_RESULT(${initdir})
 fi
 
 if test "$use_launchd" = "yes"; then
-- 
1.7.1.1

