From 0928d966e2f378efc7dc393d2664276c71f6c5a4 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 27 May 2010 15:17:20 +0400
Subject: [PATCH 01/13] Force init script installation

---
 configure.ac |   16 +++-------------
 1 files changed, 3 insertions(+), 13 deletions(-)

diff --git a/configure.ac b/configure.ac
index c609a08..cd8c807 100644
--- a/configure.ac
+++ b/configure.ac
@@ -294,19 +294,9 @@ launchd_enabled=false
 
 if test "$use_init" = "yes"; then
    AC_MSG_CHECKING(location of init directory)
-   if test -d /etc/rc.d; then
-       init_enabled=true
-       AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
-       AC_MSG_RESULT(${initdir})
-   else
-       if test -d /etc/init.d; then
-           init_enabled=true
-           AC_SUBST([initdir], ['${sysconfdir}/init.d'])
-           AC_MSG_RESULT(${initdir})
-       else
-           AC_MSG_RESULT(not found)
-       fi
-    fi
+   init_enabled=true
+   AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
+   AC_MSG_RESULT(${initdir})
 fi
 
 if test "$use_launchd" = "yes"; then
-- 
1.7.2.3

