diff -up apache-couchdb-0.9.0/bin/couchdb.tpl.in.pid apache-couchdb-0.9.0/bin/couchdb.tpl.in
--- apache-couchdb-0.9.0/bin/couchdb.tpl.in.pid	2009-05-06 19:52:22.000000000 -0300
+++ apache-couchdb-0.9.0/bin/couchdb.tpl.in	2009-05-06 19:52:59.000000000 -0300
@@ -27,7 +27,7 @@ RESPAWN_TIMEOUT=0
 DEFAULT_INI_FILE=%localconfdir%/%defaultini%
 LOCAL_INI_FILE=%localconfdir%/%localini%
 
-PID_FILE=%localstatedir%/run/couchdb.pid
+PID_FILE=%localstatedir%/run/couchdb/couchdb.pid
 
 STDOUT_FILE=couchdb.stdout
 STDERR_FILE=couchdb.stderr
