From: Peter Lemenkov <lemenkov@gmail.com>
Date: Mon, 7 Jun 2010 15:08:42 +0400
Subject: [PATCH] Remove pid-file after stop


diff --git a/bin/couchdb.tpl.in b/bin/couchdb.tpl.in
index 6ebacf4..1e1a6cf 100644
--- a/bin/couchdb.tpl.in
+++ b/bin/couchdb.tpl.in
@@ -302,6 +302,7 @@ EOF
 
 stop_couchdb () {
     PID=`_get_pid`
+    rm -f $PID_FILE
     STOP_TIMEOUT=60
     if test -n "$PID"; then
         if kill -0 $PID 2> /dev/null; then
