From 2024b34484f62475cf320aca1386f7142466329d Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 27 May 2010 15:18:29 +0400
Subject: [PATCH 02/13] Install into erllibdir by default

---
 configure |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 8d47467..239eff7 100755
--- a/configure
+++ b/configure
@@ -12193,7 +12193,7 @@ localdatadir=${datadir}/${package_identifier}
 
 localdocdir=${datadir}/doc/${package_identifier}
 
-locallibdir=${libdir}/${package_identifier}
+locallibdir=${libdir}
 
 localstatelibdir=${localstatedir}/lib/${package_identifier}
 
@@ -12209,7 +12209,7 @@ if test x${IS_WINDOWS} = xTRUE; then
     localerlanglibdir=${libdir}
 
 else
-    locallibbindir=${locallibdir}/bin
+    locallibbindir=${locallibdir}/erlang/lib/couch-${version}/priv
 
     localerlanglibdir=${locallibdir}/erlang/lib
 
-- 
1.7.2.3

