From 9bb5e5c00f5c91dcc6e0cea7d58fdf38788d9cf8 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Sun, 13 Feb 2011 14:21:20 +0300
Subject: [PATCH 3/8] More directories to search for place for init-script

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 configure.ac | 16 +++++++++++-----
 1 file changed, 11 insertions(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index a2ab2ae..f023e51 100644
--- a/configure.ac
+++ b/configure.ac
@@ -487,17 +487,23 @@ launchd_enabled=false
 
 if test "$use_init" = "yes"; then
    AC_MSG_CHECKING(location of init directory)
-   if test -d /etc/rc.d; then
+   if test -d /etc/rc.d/init.d; then
        init_enabled=true
-       AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
+       AC_SUBST([initdir], ['${sysconfdir}/rc.d/init.d'])
        AC_MSG_RESULT(${initdir})
    else
-       if test -d /etc/init.d; then
+       if test -d /etc/rc.d; then
            init_enabled=true
-           AC_SUBST([initdir], ['${sysconfdir}/init.d'])
+           AC_SUBST([initdir], ['${sysconfdir}/rc.d'])
            AC_MSG_RESULT(${initdir})
        else
-           AC_MSG_RESULT(not found)
+           if test -d /etc/init.d; then
+              init_enabled=true
+              AC_SUBST([initdir], ['${sysconfdir}/init.d'])
+              AC_MSG_RESULT(${initdir})
+           else
+              AC_MSG_RESULT(not found)
+           fi
        fi
     fi
 fi
-- 
1.8.1

