From f57afe1dc587a7b5178ee38a693837e443138c4b Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 27 May 2010 15:18:29 +0400
Subject: [PATCH 2/8] Install into erllibdir by default

---
 configure |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 7bdf280..c5f079f 100755
--- a/configure
+++ b/configure
@@ -12188,7 +12188,7 @@ localdatadir=${datadir}/${package_identifier}
 
 localdocdir=${datadir}/doc/${package_identifier}
 
-locallibdir=${libdir}/${package_identifier}
+locallibdir=${libdir}
 
 localstatelibdir=${localstatedir}/lib/${package_identifier}
 
@@ -12204,7 +12204,7 @@ if test x${IS_WINDOWS} = xTRUE; then
     localerlanglibdir=${libdir}
 
 else
-    locallibbindir=${locallibdir}/bin
+    locallibbindir=${locallibdir}/erlang/lib/couch-${version}/priv
 
     localerlanglibdir=${locallibdir}/erlang/lib
 
-- 
1.6.6.1

