From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 15 Aug 2013 18:58:25 +0400
Subject: [PATCH] Don't check for Erlang version

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>

diff --git a/configure.ac b/configure.ac
index fe4f79a..08b8c42 100644
--- a/configure.ac
+++ b/configure.ac
@@ -417,14 +417,6 @@ major_version=`echo $version | ${AWK} "{print \\$1}"`
 minor_version=`echo $version | ${AWK} "{print \\$2}"`
 patch_version=`echo $version | ${AWK} "{print \\$3}"`
 
-if test $major_version -ne 5; then
-    as_fn_error $? "$erlang_version_error" "$LINENO" 5
-fi
-
-if test $minor_version -lt 8 -o $minor_version -gt 10; then
-    as_fn_error $? "$erlang_version_error" "$LINENO" 5
-fi
-
 AC_MSG_RESULT(compatible)
 
 otp_release="`\
