From ecb89a85b6849f1ee7d5df09dfb40fdac87194c2 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Sun, 13 Feb 2011 13:52:38 +0300
Subject: [PATCH 01/14] Do not gzip doc-files and do not install installation
 instructions

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 Makefile.am | 57 +++++++++------------------------------------------------
 1 file changed, 9 insertions(+), 48 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 8703a30..75b9d35 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -15,18 +15,15 @@ SUBDIRS = bin etc src share test var utils
 ACLOCAL_AMFLAGS = -I m4
 
 localdoc_DATA = \
-    AUTHORS.gz \
-    BUGS.gz \
-    CHANGES.gz \
-    DEVELOPERS.gz \
-    INSTALL.gz \
-    INSTALL.Unix.gz \
-    INSTALL.Windows.gz \
-    LICENSE.gz \
-    NEWS.gz \
-    NOTICE.gz \
-    README.gz \
-    THANKS.gz
+    AUTHORS \
+    BUGS \
+    CHANGES \
+    DEVELOPERS \
+    LICENSE \
+    NEWS \
+    NOTICE \
+    README \
+    THANKS
 
 DISTCLEANFILES = $(localdoc_DATA)
 
@@ -48,42 +45,6 @@ EXTRA_DIST = \
     build-aux/sphinx-touch \
     license.skip
 
-AUTHORS.gz: AUTHORS
-	gzip -9 < $< > $@
-
-BUGS.gz: BUGS
-	gzip -9 < $< > $@
-
-CHANGES.gz: CHANGES
-	gzip -9 < $< > $@
-
-DEVELOPERS.gz: DEVELOPERS
-	gzip -9 < $< > $@
-
-INSTALL.gz: INSTALL
-	gzip -9 < $< > $@
-
-INSTALL.Unix.gz: INSTALL.Unix
-	gzip -9 < $< > $@
-
-INSTALL.Windows.gz: INSTALL.Windows
-	gzip -9 < $< > $@
-
-LICENSE.gz: LICENSE
-	gzip -9 < $< > $@
-
-NEWS.gz: NEWS
-	gzip -9 < $< > $@
-
-NOTICE.gz: NOTICE
-	gzip -9 < $< > $@
-
-README.gz: README.rst
-	gzip -9 < $< > $@
-
-THANKS.gz: THANKS
-	gzip -9 < $< > $@
-
 check: dev check-js
 if TESTS
 	$(top_builddir)/test/etap/run $(top_srcdir)/test/etap
-- 
1.8.3.1

