From 4d79cfe314d2b08a46cf25460402e8d7db788576 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 26 Aug 2010 13:07:33 +0400
Subject: [PATCH 10/11] Do not install gzipped docs

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 Makefile.am |   52 +-----------------------------------
 Makefile.in |   85 +++++------------------------------------------------------
 2 files changed, 8 insertions(+), 129 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 8e16baf..06f7be9 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -14,21 +14,7 @@ SUBDIRS = bin etc src share test var utils
 
 ACLOCAL_AMFLAGS = -I m4
 
-localdoc_DATA = \
-    AUTHORS.gz \
-    BUGS.gz \
-    CHANGES.gz \
-    DEVELOPERS.gz \
-    INSTALL.gz \
-    INSTALL.Unix.gz \
-    INSTALL.Windows.gz \
-    LICENSE.gz \
-    NEWS.gz \
-    NOTICE.gz \
-    README.gz \
-    THANKS.gz
-
-DISTCLEANFILES = $(localdoc_DATA)
+DISTCLEANFILES =
 
 EXTRA_DIST = \
     AUTHORS \
@@ -45,42 +31,6 @@ EXTRA_DIST = \
     THANKS \
     license.skip
 
-AUTHORS.gz: $(top_srcdir)/AUTHORS
-	-gzip -9 < $< > $@
-
-BUGS.gz: $(top_srcdir)/BUGS
-	-gzip -9 < $< > $@
-
-CHANGES.gz: $(top_srcdir)/CHANGES
-	-gzip -9 < $< > $@
-
-DEVELOPERS.gz: $(top_srcdir)/DEVELOPERS
-	-gzip -9 < $< > $@
-
-INSTALL.gz: $(top_srcdir)/INSTALL
-	-gzip -9 < $< > $@
-
-INSTALL.Unix.gz: $(top_srcdir)/INSTALL.Unix
-	-gzip -9 < $< > $@
-
-INSTALL.Windows.gz: $(top_srcdir)/INSTALL.Windows
-	-gzip -9 < $< > $@
-
-LICENSE.gz: $(top_srcdir)/LICENSE
-	-gzip -9 < $< > $@
-
-NEWS.gz: $(top_srcdir)/NEWS
-	-gzip -9 < $< > $@
-
-NOTICE.gz: $(top_srcdir)/NOTICE
-	-gzip -9 < $< > $@
-
-README.gz: $(top_srcdir)/README
-	-gzip -9 < $< > $@
-
-THANKS.gz: $(top_srcdir)/THANKS
-	-gzip -9 < $< > $@
-
 check: dev
 	$(top_builddir)/test/etap/run
 
diff --git a/Makefile.in b/Makefile.in
index 0e9f10a..bc76e26 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -85,7 +85,7 @@ am__base_list = \
   sed '$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;$$!N;s/\n/ /g' | \
   sed '$$!N;$$!N;$$!N;$$!N;s/\n/ /g'
 am__installdirs = "$(DESTDIR)$(localdocdir)"
-DATA = $(localdoc_DATA)
+DATA =
 RECURSIVE_CLEAN_TARGETS = mostlyclean-recursive clean-recursive	\
   distclean-recursive maintainer-clean-recursive
 AM_RECURSIVE_TARGETS = $(RECURSIVE_TARGETS:-recursive=) \
@@ -290,21 +290,8 @@ version_revision = @version_revision@
 version_stage = @version_stage@
 SUBDIRS = bin etc src share test var utils
 ACLOCAL_AMFLAGS = -I m4
-localdoc_DATA = \
-    AUTHORS.gz \
-    BUGS.gz \
-    CHANGES.gz \
-    DEVELOPERS.gz \
-    INSTALL.gz \
-    INSTALL.Unix.gz \
-    INSTALL.Windows.gz \
-    LICENSE.gz \
-    NEWS.gz \
-    NOTICE.gz \
-    README.gz \
-    THANKS.gz
-
-DISTCLEANFILES = $(localdoc_DATA)
+
+DISTCLEANFILES =
 EXTRA_DIST = \
     AUTHORS \
     BUGS \
@@ -384,27 +371,6 @@ clean-libtool:
 
 distclean-libtool:
 	-rm -f libtool config.lt
-install-localdocDATA: $(localdoc_DATA)
-	@$(NORMAL_INSTALL)
-	test -z "$(localdocdir)" || $(MKDIR_P) "$(DESTDIR)$(localdocdir)"
-	@list='$(localdoc_DATA)'; test -n "$(localdocdir)" || list=; \
-	for p in $$list; do \
-	  if test -f "$$p"; then d=; else d="$(srcdir)/"; fi; \
-	  echo "$$d$$p"; \
-	done | $(am__base_list) | \
-	while read files; do \
-	  echo " $(INSTALL_DATA) $$files '$(DESTDIR)$(localdocdir)'"; \
-	  $(INSTALL_DATA) $$files "$(DESTDIR)$(localdocdir)" || exit $$?; \
-	done
-
-uninstall-localdocDATA:
-	@$(NORMAL_UNINSTALL)
-	@list='$(localdoc_DATA)'; test -n "$(localdocdir)" || list=; \
-	files=`for p in $$list; do echo $$p; done | sed -e 's|^.*/||'`; \
-	test -n "$$files" || exit 0; \
-	echo " ( cd '$(DESTDIR)$(localdocdir)' && rm -f" $$files ")"; \
-	cd "$(DESTDIR)$(localdocdir)" && rm -f $$files
-
 # This directory's subdirectories are mostly independent; you can cd
 # into them and run `make' without going through this Makefile.
 # To change the values of `make' variables: instead of editing Makefiles,
@@ -774,7 +740,7 @@ info: info-recursive
 
 info-am:
 
-install-data-am: install-localdocDATA
+install-data-am:
 	@$(NORMAL_INSTALL)
 	$(MAKE) $(AM_MAKEFLAGS) install-data-hook
 install-dvi: install-dvi-recursive
@@ -821,7 +787,7 @@ ps: ps-recursive
 
 ps-am:
 
-uninstall-am: uninstall-localdocDATA
+uninstall-am:
 
 .MAKE: $(RECURSIVE_CLEAN_TARGETS) $(RECURSIVE_TARGETS) all \
 	ctags-recursive install-am install-data-am install-strip \
@@ -837,50 +803,13 @@ uninstall-am: uninstall-localdocDATA
 	html-am info info-am install install-am install-data \
 	install-data-am install-data-hook install-dvi install-dvi-am \
 	install-exec install-exec-am install-html install-html-am \
-	install-info install-info-am install-localdocDATA install-man \
+	install-info install-info-am install-man \
 	install-pdf install-pdf-am install-ps install-ps-am \
 	install-strip installcheck installcheck-am installdirs \
 	installdirs-am maintainer-clean maintainer-clean-generic \
 	mostlyclean mostlyclean-generic mostlyclean-libtool pdf pdf-am \
-	ps ps-am tags tags-recursive uninstall uninstall-am \
-	uninstall-localdocDATA
-
-
-AUTHORS.gz: $(top_srcdir)/AUTHORS
-	-gzip -9 < $< > $@
-
-BUGS.gz: $(top_srcdir)/BUGS
-	-gzip -9 < $< > $@
-
-CHANGES.gz: $(top_srcdir)/CHANGES
-	-gzip -9 < $< > $@
-
-DEVELOPERS.gz: $(top_srcdir)/DEVELOPERS
-	-gzip -9 < $< > $@
-
-INSTALL.gz: $(top_srcdir)/INSTALL
-	-gzip -9 < $< > $@
-
-INSTALL.Unix.gz: $(top_srcdir)/INSTALL.Unix
-	-gzip -9 < $< > $@
-
-INSTALL.Windows.gz: $(top_srcdir)/INSTALL.Windows
-	-gzip -9 < $< > $@
-
-LICENSE.gz: $(top_srcdir)/LICENSE
-	-gzip -9 < $< > $@
-
-NEWS.gz: $(top_srcdir)/NEWS
-	-gzip -9 < $< > $@
-
-NOTICE.gz: $(top_srcdir)/NOTICE
-	-gzip -9 < $< > $@
-
-README.gz: $(top_srcdir)/README
-	-gzip -9 < $< > $@
+	ps ps-am tags tags-recursive uninstall uninstall-am
 
-THANKS.gz: $(top_srcdir)/THANKS
-	-gzip -9 < $< > $@
 
 check: dev
 	$(top_builddir)/test/etap/run
-- 
1.7.2.1

