From 1de07dd7ccb0696a59306fc58d73231dcbbfb367 Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Fri, 15 Mar 2013 16:41:59 +0400
Subject: [PATCH 09/10] Mostly cosmetic - proplist ordering in R16B

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 test/etap/120-stats-collect.t | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/etap/120-stats-collect.t b/test/etap/120-stats-collect.t
index dee8876..44744d6 100755
--- a/test/etap/120-stats-collect.t
+++ b/test/etap/120-stats-collect.t
@@ -125,13 +125,13 @@ test_all() ->
     couch_stats_collector:record(bar, 1.0),
     etap:is(
         couch_stats_collector:all(),
-        [{foo, 0}, {hoopla, 0}, {bar, [1.0, 0.0]}],
+        [{hoopla,0},{foo,0},{bar,[1.0,0.0]}],
         "all/0 returns all counters and absolute values."
     ),
     
     etap:is(
         couch_stats_collector:all(incremental),
-        [{foo, 0}, {hoopla, 0}],
+        [{hoopla,0},{foo,0}],
         "all/1 returns only the specified type."
     ),
     
-- 
1.8.2.1

