--- src/Makefile.am	2009-09-30 07:11:03.000000000 +0400
+++ src/Makefile.am	2010-05-14 11:22:53.861117171 +0400
@@ -10,4 +10,4 @@
 ## License for the specific language governing permissions and limitations under
 ## the License.
 
-SUBDIRS = couchdb erlang-oauth etap ibrowse mochiweb
+SUBDIRS = couchdb etap ibrowse mochiweb
--- src/Makefile.in	2010-04-11 17:54:10.000000000 +0400
+++ src/Makefile.in	2010-05-14 11:22:43.407116852 +0400
@@ -239,7 +239,7 @@
 version_release = @version_release@
 version_revision = @version_revision@
 version_stage = @version_stage@
-SUBDIRS = couchdb erlang-oauth etap ibrowse mochiweb
+SUBDIRS = couchdb etap ibrowse mochiweb
 all: all-recursive
 
 .SUFFIXES:
--- configure~	2010-04-11 17:54:12.000000000 +0400
+++ configure	2010-05-14 11:38:01.635117281 +0400
@@ -12137,8 +12137,6 @@
 
 ac_config_files="$ac_config_files src/couchdb/priv/Makefile"
 
-ac_config_files="$ac_config_files src/erlang-oauth/Makefile"
-
 ac_config_files="$ac_config_files src/etap/Makefile"
 
 ac_config_files="$ac_config_files src/ibrowse/Makefile"
@@ -13160,7 +13158,6 @@
     "src/couchdb/couch.app.tpl") CONFIG_FILES="$CONFIG_FILES src/couchdb/couch.app.tpl" ;;
     "src/couchdb/Makefile") CONFIG_FILES="$CONFIG_FILES src/couchdb/Makefile" ;;
     "src/couchdb/priv/Makefile") CONFIG_FILES="$CONFIG_FILES src/couchdb/priv/Makefile" ;;
-    "src/erlang-oauth/Makefile") CONFIG_FILES="$CONFIG_FILES src/erlang-oauth/Makefile" ;;
     "src/etap/Makefile") CONFIG_FILES="$CONFIG_FILES src/etap/Makefile" ;;
     "src/ibrowse/Makefile") CONFIG_FILES="$CONFIG_FILES src/ibrowse/Makefile" ;;
     "src/mochiweb/Makefile") CONFIG_FILES="$CONFIG_FILES src/mochiweb/Makefile" ;;
--- configure.ac~	2009-11-12 06:25:31.000000000 +0300
+++ configure.ac	2010-05-14 11:40:25.522117742 +0400
@@ -335,7 +335,6 @@
 AC_CONFIG_FILES([src/couchdb/couch.app.tpl])
 AC_CONFIG_FILES([src/couchdb/Makefile])
 AC_CONFIG_FILES([src/couchdb/priv/Makefile])
-AC_CONFIG_FILES([src/erlang-oauth/Makefile])
 AC_CONFIG_FILES([src/etap/Makefile])
 AC_CONFIG_FILES([src/ibrowse/Makefile])
 AC_CONFIG_FILES([src/mochiweb/Makefile])
