From 81edc422be58099e832f5d81cf776b8fc964785f Mon Sep 17 00:00:00 2001
From: Peter Lemenkov <lemenkov@gmail.com>
Date: Thu, 26 Aug 2010 13:22:56 +0400
Subject: [PATCH 8/8] Change respawn timeout to 0.

Signed-off-by: Peter Lemenkov <lemenkov@gmail.com>
---
 etc/default/couchdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/default/couchdb b/etc/default/couchdb
index c2a3f2a..f589c0a 100644
--- a/etc/default/couchdb
+++ b/etc/default/couchdb
@@ -3,5 +3,5 @@
 COUCHDB_USER=couchdb
 COUCHDB_STDOUT_FILE=/dev/null
 COUCHDB_STDERR_FILE=/dev/null
-COUCHDB_RESPAWN_TIMEOUT=5
+COUCHDB_RESPAWN_TIMEOUT=0
 COUCHDB_OPTIONS=
-- 
1.7.12.1

